android/gradle/dependencies.gradle

```diff
@@ -76,8 +76,8 @@ def androidx = [
 def test = [
     junit: "junit:junit:4.12",
     mockito: "org.mockito:mockito-core:2.0.42-beta",
-    assertj: "org.assertj:assertj-core:2.6.0",
     compileTesting: "com.google.testing.compile:compile-testing:0.17",
+    truth: "com.google.truth:truth:0.43",
```

Comment:
Truth is at 1.0. Are you keeping at 0.43 since it's an easier migration (without breaking changes)?

Reply:
matching internal version for now. will follow up w/ library bumps in a separate PR